src/js/utilities/characterSort.spec.js

```diff
@@ -0,0 +1,29 @@
+import { getSortedCharacters } from './characterSort';
+
+describe('characterSort.js', () => {
+  it('Should return the same back.', () => {
```

Comment:
When writing tests, I like to be as clear a possible as to what is being tested.  This is especially helpful if a test fails and another developer has to troubleshoot.

I think here, your `it` statement could be more descriptive of what is being tested.  Maybe something like "should return an unsorted character array" would work.  (Doesn't have to be this exact wording.)